.rubocop.yml

```diff
@@ -0,0 +1,3 @@
+Lint/AmbiguousBlockAssociation:
```

Comment:
I feel like I ran into this the other day and managed to fix it. I'll have a quick play to see if I can?

Reply:
Ah, I see the issue you referenced in commit message. Alright, this seems fine to me.